core/common/src/test/java/alluxio/heartbeat/HeartbeatThreadTest.java

```diff
@@ -76,9 +79,11 @@ public static void afterClass() {
    */
   @Test
   public void serialHeartbeatThreadTest() throws Exception {
-    Thread thread = new DummyHeartbeatTestThread();
+    FutureTask<Null> task = new FutureTask<>(new DummyHeartbeatTestCallable());
```

Comment:
is there a difference between using `org.apache.commons.lang3.ObjectUtils.Null` and `java.lang.Void`? I have seen `Void` being used in other part of our codebase and I would prefer using that here if it is possible